src/com/temporaryteam/noticeditor/controller/NoticeSettingsController.java

```diff
@@ -41,28 +42,33 @@ public void setNoticeController(NoticeController noticeController) {
 	 */
 	@Override
 	public void initialize(URL url, ResourceBundle rb) {
-		choiceBoxNoticeStatus.setItems(FXCollections.observableArrayList(
-				rb.getString("normal"), rb.getString("important")
-		));
-		choiceBoxNoticeStatus.getSelectionModel().selectedIndexProperty().addListener(new ChangeListener<Number>() {
-
+		NoticeStatus.add(rb.getString("normal"));
+		NoticeStatus.add(rb.getString("important"));
+		NoticeStatus.save();
+		
+		choiceBoxNoticeStatus.setItems(NoticeStatus.asObservable());
+		choiceBoxNoticeStatus.getSelectionModel().selectedItemProperty().addListener(new ChangeListener<String>() {
 			@Override
-			public void changed(ObservableValue<? extends Number> observable, Number oldValue, Number newValue) {
+			public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
 				NoticeTreeItem currentNotice = noticeController.getCurrentNotice();
-				if (currentNotice != null && currentNotice.isLeaf()) {
-					currentNotice.setStatus(newValue.intValue());
+				if (null != currentNotice && newValue != null && currentNotice.isLeaf()) {
```

Comment:
Replace to `if ((currentNotice != null) && (newValue != null) && currentNotice.isLeaf()) ` please